robocopkaka/bootcamp-29-project
# #158212525 Grant users token on signup

#### What does this PR do?
Grant users tokens when they signup
#### Description of Task to be completed?
* add generating token functionality to the `create` method in the users controller
* set token in `sessionStorage` in `registerActions`
* modify `registerReducer` to make use of the changes made in `registerActions`
* add keys for jwt to the `register` object in initial state
* modify tests for `registerActions` and `registerReducer`
* set `loggedIn` prop in `Header` component based on the jwt value in either the `register` or `session` object in state
#### How should this be manually tested?
* clone this repo
* cd into the project folder
* run npm install to install all dependecies
* run npm run start:all to start up the application
* go to localhost:8080 on your browser
* click on Signup
* enter your details
* click on Signup
* request for admin access
* click on Centers on the navbar
* click on the red add button at the bottom right corner
* a modal with the form used for adding centers should come up
* enter the details of the center
* click on Add Center
* the modal should close and you should see the new center on the page
#### Any background context you want to provide?
you have to refresh the page after logging out to remove the `Profile` link. bug to be fixed in another branch
#### What are the relevant pivotal tracker stories?(if applicable)
#158212525
